FMB/FiniteModelBuilder.cpp

```diff
@@ -1601,9 +1601,6 @@ MainLoopResult FiniteModelBuilder::runImpl()
       }
       cout << "]" << endl;
     }
-    Timer::syncClock();
-    if(env.timeLimitReached()){ return MainLoopResult(Statistics::TIME_LIMIT); }
-
```

Comment:
Lots of code like this. I don't like to remove code that I don't understand, but as far as I can tell this was pointless once we had signal-based timeouts.